tests/generators/ssz_generic/ssz_container.py

```diff
@@ -104,17 +104,16 @@ def invalid_cases():
                          RandomizationMode.mode_nil_count,
                          RandomizationMode.mode_one_count,
                          RandomizationMode.mode_max_count]:
-                if len(offsets) != 0:
```

Comment:
`len(offsets) == 0` always as we already checked to enter into this flow.

and we do not mutate them while iterating over the hardcoded list of modes...